docs/resources/apig_vpc_channel.md

```diff
@@ -38,79 +38,88 @@ resource "huaweicloud_apig_vpc_channel" "test" {
 
 The following arguments are supported:
 
-* `region` - (Optional, String, ForceNew) Specifies the region in which to create the VPC channel resource.
-  If omitted, the provider-level region will be used.
-  Changing this will create a new VPC channel resource.
+* `region` - (Optional, String, ForceNew) Specifies the region where the dedicated instance is located.  
```

Comment:
s/dedicated instance/VPC channel

Reply:
the vpc channel is belongs to dedicated instance